RIOT-OS/RIOT
# boards/arduino-nano: Support for the Arduino Nano

### Contribution description

This PR adds support for the Arduino Nano. The Arduino Nano board is the cheapest member of the Arduino family and used the same MCU as the Arduino Uno. It differs in the form factor (the Nano is much smaller), it uses an integrated FT232RL TTL adapter instead of an ATmega16u2 to provide access to the serial console via USB, and it uses a different bootloader (which occupies 2 KiB of the 32 KiB flash instead of 0.5 KiB occupied on the Arduino Uno). This commit mostly copy pastes code from the Arduino Uno.  

### Testing procedure

- Check the provided documentation
- Check if `make BOARD=arduino-nano flash` and `make BOARD=arduino-nano term` works for `examples/hello-world` 

As this shares most code with the Arduino UNO, no huge troubles are expected.

### Issues/PRs references

None
